src/test/testRunner.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path';
+import * as cp from 'child_process';
+import {
+  runTests,
+  downloadAndUnzipVSCode,
+  resolveCliPathFromVSCodeExecutablePath,
+} from 'vscode-test';
+
+async function main(): Promise<void> {
+  try {
+    const executable = await downloadAndUnzipVSCode();
+    const cliPath = resolveCliPathFromVSCodeExecutablePath(executable);
+
+    // Install the latest released redhat.ansible extension
+    const installLog = cp.execSync(
+      `"${cliPath}" --install-extension redhat.ansible --force`
+    );
+    console.log(installLog.toString());
+
+    // Install the dependant extensions
```

Comment:
Why can't this just come from the deps?

Reply:
Sorry, unable to understand it. Could you please elaborate?